pkg/controller/servicemesh/controlplane/pruner.go

```diff
@@ -31,7 +31,7 @@ var (
 		gvk("apps", "v1", "Deployment"),
 		gvk("apps", "v1", "DaemonSet"),
 		gvk("apps", "v1", "StatefulSet"),
-		gvk("extensions", "v1beta1", "Ingress"),
+		gvk("networking.k8s.io", "v1", "Ingress"),
```

Comment:
should we remove beta resources from this list? or just add the v1 to it so both end up pruned?